{
  "arch": "x86_64",
  "build": "h3f2d84a_0",
  "build_number": 0,
  "channel": "https://conda.anaconda.org/conda-forge/noarch/",
  "constrains": [],
  "depends": [
    "__glibc >=2.17,<3.0.a0",
    "libgcc >=13",
    "libstdcxx >=13"
  ],
  "fn": "libgpg-error-1.55-h3f2d84a_0.conda",
  "license": "LGPL-2.1-only",
  "md5": "2bd47db5807daade8500ed7ca4c512a4",
  "name": "libgpg-error",
  "platform": "linux",
  "sha256": "697334de4786a1067ea86853e520c64dd72b11a05137f5b318d8a444007b5e60",
  "size": 312184,
  "subdir": "linux-64",
  "timestamp": 1745575272000,
  "url": "https://conda.anaconda.org/conda-forge/linux-64/libgpg-error-1.55-h3f2d84a_0.conda",
  "version": "1.55"
}